composer.json

```diff
@@ -30,7 +30,7 @@
         "openeuropa/behat-transformation-context": "~0.1",
         "openeuropa/code-review": "~1.0.0-beta4",
         "openeuropa/drupal-core-require-dev": "^8.7",
-        "openeuropa/oe_content": "dev-OPENEUROPA-2221-2",
+        "openeuropa/oe_content": "dev-OPENEUROPA-2771",
```

Comment:
We need to use here `dev-OPENEUROPA-2221-3` or, if that's merged, `dev-EPIC-Event`.